tasks/RedundantRBSSelectorTask.cpp

```diff
@@ -194,6 +208,29 @@ void RedundantRBSSelectorTask::updateHook()
         _rbs_out.write(secondary_rbs);
     }
 }
+
+bool RedundantRBSSelectorTask::areOrientationDivergent(Orientation first,
+    Orientation second)
+{
+    Vector3d diff_orientation = getEuler(first) - getEuler(second);
```

Comment:
This not seen to be right. Test an edge case (3.14, -3.14)
I would recommend to create a base::Angle from the euler angles, and use the - operator already implemented.